ytools/y2tool/rubocop_yast_style.yml

```diff
@@ -88,9 +113,63 @@ Style/CaseIndentation:
 Style/NegatedIf:
   Enabled: false
 
-# allow more than 10 lines for methods
+# allow more than 10 lines for methods as some team member feel unconfortable with it
 Metrics/MethodLength:
   Max: 30
 
+# allow more than 100 lines for class as some team member feel unconfortable with it
 Metrics/ClassLength:
   Max: 250
+
+# YaST use older rubocop that do not have allow predicates as default to true,
+# so explicitelly allow it for older ones
+Style/TrivialAccessors:
+ # predicates cannot be done with accessors
+ AllowPredicates: true
+
+# Allow using and/or for driving code flow as its original intention.
+# Forbid it only in conditionals
+# So this rule allow something like
+# a = action_a or raise "Cannot do a"
+Style/AndOr:
+  EnforcedStyle: conditionals
+
+# Access modified afect globally all following method definition, so
+# it deserve bigger visibility then hiddin in method definition block.
+# for that reason we found better style:
+# class C
+#   trash
+#   ....
+#   ...
+#
+# private
+#
+#   ....
+#   trash
+#   ...
+# end
+#
+# then style where access modifier can be easier to overlook in longer class:
+# class C
+#   trash
+#   ....
+#   ...
+#
+#   private
+#
+#   ....
+#   trash
+#   ...
+# end
+Style/AccessModifierIndentation:
+  EnforcedStyle: outdent
```

Comment:
So this is the one change that will make other packages fail, right? Will you fix them all?